libs/arrow-meta/src/main/kotlin/arrow/meta/phases/analysis/diagnostic/PositionStrategies.kt

```diff
@@ -1,58 +0,0 @@
-package arrow.meta.phases.analysis.diagnostic
-
-import org.jetbrains.kotlin.builtins.StandardNames
-import org.jetbrains.kotlin.com.intellij.openapi.util.TextRange
-import org.jetbrains.kotlin.com.intellij.psi.PsiElement
-import org.jetbrains.kotlin.diagnostics.ParametrizedDiagnostic
-import org.jetbrains.kotlin.diagnostics.PositioningStrategy
-import org.jetbrains.kotlin.diagnostics.hasSyntaxErrors
-import org.jetbrains.kotlin.diagnostics.markElement
-import org.jetbrains.kotlin.psi.KtAnnotationEntry
-import org.jetbrains.kotlin.psi.KtDeclaration
-import org.jetbrains.kotlin.psi.KtObjectDeclaration
-import org.jetbrains.kotlin.psi.psiUtil.textRangeWithoutComments
-import org.jetbrains.kotlin.resolve.BindingContext
-
-/**
```

Comment:
@i-walker we did not find references to any of the functions on this file. Tagging you here to verify we can get rid of this and this is not used somewhere else.

Reply:
Yes that is fine, these were some left overs from the ide implementation of proofs :)